scanpy/_utils/__init__.py

```diff
@@ -20,10 +22,10 @@
 import numpy as np
 from anndata import AnnData
 from anndata import __version__ as anndata_version
-from numpy import random
 from numpy.typing import NDArray
 from packaging import version
 from scipy import sparse
+from sklearn.dummy import check_random_state
```

Comment:
Isn’t this the exported location of this?

```suggestion
from sklearn.utils import check_random_state
```